include/fluent-bit/flb_config.h

```diff
@@ -243,7 +243,7 @@ struct flb_config {
      */
     uint16_t in_table_id[512];
 
-    void *sched;
+    struct flb_sched *sched;
```

Comment:
Wonder why this is a `void *` in the first place. Seems like all pointers are `void *` in this file. I'd keep it `void *` unless we understand this convention and have a reason to break it.

Reply:
void * is useful if at compile time it could store multiple different structs. Here its always one struct, so there's no good reason AFAIK for it to be void.